Migrated from UIWebView to WKWebView

TripwireNL opened the pull request:
Since Apple is starting to reject apps using (or referring to) UIWebView I've removed the UIWebView+Extensions class en UIWebViewExtensionTests and created the WKWebView+Extensions and WKWebViewExtensionTests so BFKit-Swift doesn't refer to UIWebView anymore.

See also:
https://github.com/AFNetworking/AFNetworking/issues/4428
https://github.com/react-native-community/react-native-webview/issues/819

Commit:
Migrated from UIWebView to WKWebView

Since Apple is starting to reject apps using (or referring to) UIWebView I've removed the UIWebView+Extensions class en UIWebViewExtensionTests and created the WKWebView+Extensions and WKWebViewExtensionTests so BFKit-Swift doesn't refer to UIWebView anymore.

FabrizioBrancati commented on Sources/BFKit/Apple/UIKit/WKWebView+Extensions.swift:
```diff
@@ -1,6 +1,6 @@
-//  UIWebView+Extensions.swift
-//  BFKit-Swift
+//  WKWebView+Extensions.swift
+//  BFKit iOS
```
It should be `BFKit-Swift`

FabrizioBrancati commented on Sources/BFKit/Apple/UIKit/WKWebView+Extensions.swift:
```diff
@@ -26,21 +26,21 @@
+  /// Load the requested website.
+  ///
+  /// - Parameter website: Website to load
+  func load(Website website: String) {
```
If we want to insert the `Website` keyword inside the parenthesis, you need to add it in lowercase (`website`)

FabrizioBrancati commented on Tests/BFKitTests/Apple/UIKit/WKWebViewExtensionTests.swift:
```diff
@@ -0,0 +1,42 @@
+//
```
This is the right header, even if we are adding tests here:
```
//	
//  UIWebViewExtensionTests.swift	
//  BFKit-Swift	
//	
//  The MIT License (MIT)	
//	
//  Copyright (c) 2015 - 2018 Fabrizio Brancati.	
//	
//  Permission is hereby granted, free of charge, to any person obtaining a copy	
//  of this software and associated documentation files (the "Software"), to deal	
//  in the Software without restriction, including without limitation the rights	
//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell	
//  copies of the Software, and to permit persons to whom the Software is	
//  furnished to do so, subject to the following conditions:	
//	
//  The above copyright notice and this permission notice shall be included in all	
//  copies or substantial portions of the Software.	
//	
//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR	
//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,	
//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE	
//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER	
//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,	
//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE	
//  SOFTWARE.
```

FabrizioBrancati reviewed (changes requested):
Thanks for the PR! I would really love to have it inside BFKit-Swift!

I just left some comments on the code, can you fix them? 😄

Commit:
Small fixes

TripwireNL commented:
@FabrizioBrancati Thanks for the feedback, I've changed the comments you had :) Though I don't agree with the:
`func load(Website website: String)` 

To me this seems the correct way! Anyways I removed the Website keyword and replaced it with _

Commit:
Changed header to WKWebViewExtensionTests

FabrizioBrancati commented on Sources/BFKit/Apple/UIKit/WKWebView+Extensions.swift:
```diff
@@ -26,21 +26,21 @@
+  /// Load the requested website.
+  ///
+  /// - Parameter website: Website to load
+  func load(_ website: String) {
```
Sorry for the misunderstanding, I meant to change `Website` to `website`, so the right signature would be `load(webiste website: String)`

TripwireNL replied:
@FabrizioBrancati Ah like that let me change that for you xD

TripwireNL replied:
@FabrizioBrancati Actually when I change it to:
`load(website website: String)` 
Swift throws a warning: Extraneous duplicate parameter name; 'website' already has an argument label

That's why I made it:
`load(Website website: String)`

Anyways I left it like the original:
`func load(_ website: String)`

FabrizioBrancati replied:
Oops sorry, the right signature would be `load(website: String)`, because with that the parameter will be the same of the signature.

Last change, before accepting the PR and releasing a new (breaking) version 😉

TripwireNL replied:
Done :)

FabrizioBrancati commented on Tests/BFKitTests/Apple/UIKit/WKWebViewExtensionTests.swift:
```diff
@@ -27,25 +27,34 @@
 @testable import BFKit
 import Foundation
 import UIKit
```
We don't need to import UIKit anymore

FabrizioBrancati reviewed (changes requested):
There has been a misunderstanding about the `load` signature 😄

Commit:
Removed unnecessary import

Commit:
Small fix
